thermalnetwork/network.py

```diff
@@ -748,19 +749,44 @@ def run_sizer_from_cli_worker(
     network.total_network_pipe_length = total_network_length
 
     loop_order_list = []
+    feature_group = defaultdict(list)
+
+    # Source type to field name
+    source_types = {
+        "Ground Heat Exchanger": "list_ghe_ids_in_group",
+        "Central Ambient Water": "list_source_ids_in_group",
```

Comment:
Is waste heat source categorized as `Central Ambient Water` in the geojson file?